native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/AppDelegate.m

```diff
@@ -157,8 +156,8 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
     //Uncomment following block to enable IDP Login flow
     /*
     return [[SFUserAccountManager sharedInstance] handleAdvancedAuthenticationResponse:url options:options];
-    */
-   return NO;
+    // */ return NO;
```

Comment:
The commenting out here is kinda weird. Can we either remove the extra `//` or `/* */`?